packages/components/src/Badge/Badge.tsx

```diff
@@ -0,0 +1,27 @@
+import React, { useCallback } from 'react';
+
+import { createCssClassNames } from '@ibexa/ids-core/helpers/cssClassNames';
+
+import { BadgeProps, BadgeSize } from './Badge.types';
+
+const MAX_BADGE_VALUE = 99;
+const THRESHOLD = {
+    [BadgeSize.Medium]: 100,
+    [BadgeSize.Small]: 10,
+};
+const Badge = ({ className = '', size = BadgeSize.Medium, value }: BadgeProps) => {
+    const isWide = value >= THRESHOLD[size];
+    const componentClassName = createCssClassNames({
+        'ids-badge': true,
+        [`ids-badge--${size}`]: true,
+        'ids-badge--wide': isWide,
+        [className]: !!className,
+    });
+    const formatBadgeValue = useCallback((val: number): string => {
```

Comment:
we avoid abbreviations in variables names.
```suggestion
    const formatBadgeValue = useCallback((value: number): string => {
```